taxonomy: Translations for the additives taxonomy

teolemon opened the pull request:
<!-- IMPORTANT CHECKLIST
Make sure you've done all the following (You can delete the checklist before submitting)
- [ ] PR title is prefixed by one of the following: feat, fix, docs, style, refactor, test, build, ci, chore, revert, l10n, taxonomy
- [ ] Code is well documented
- [ ] Include unit tests for new functionality
- [ ] Code passes GitHub workflow checks in your branch
- [ ] If you have multiple commits please combine them into one commit by squashing them.
- [ ] Read and understood the [contribution guidelines](https://github.com/openfoodfacts/openfoodfacts-server/blob/main/CONTRIBUTING.md)
-->
### What
- taxonomy: Translate additives

teolemon committed:
Update additives.txt

teolemon commented on taxonomies/additives.txt:
```diff
@@ -9523,27 +9519,27 @@ wikidata:en:Q45044
+el:E349, Αμμωνιακά μαλικά
+es:E349, Malatos de amonio
+et:E349, Ammooniummalaadid
+fi:E349, Ammoniummalaatit
```
```suggestion
fi:E349, Ammoniummalaatti, Ammoniummalaattia
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -9523,27 +9519,27 @@ wikidata:en:Q45044
+da:E349, Ammoniummalater
+de:E349, Ammoniummalate
+el:E349, Αμμωνιακά μαλικά
+es:E349, Malatos de amonio
```
```suggestion
es:E349, Malato de amonio, Malato amónico, Malatos de amonio
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -9523,27 +9519,27 @@ wikidata:en:Q45044
+et:E349, Ammooniummalaadid
+fi:E349, Ammoniummalaatit
+fr:E349, Malates d'ammonium
+hu:E349, Ammónium-malátok
```
```suggestion
hu:E349, Ammónium-malát, Ammónium-malátok
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -9552,28 +9548,28 @@ wikidata:en:Q4747303
-et:E350, E350 food additive
-fi:E350, natriummalaatit
-fr:E350, malates de sodium, malates
+el:E350, Μαλικά νάτρια
```
```suggestion
el:E350, άλατα του μηλικού οξέος με νάτριο, Μαλικά νάτρια
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -9970,27 +9966,27 @@ wikidata:en:Q213050
+es:E364, Succinatos de sodio
+et:E364, Naatriumsuktsinaadid
+fi:E364, Natriumsukkinaatit
+fr:E364, Succinates de sodium
```
```suggestion
fr:E364, Succinates de sodium, Succinate monosodique, Succinate disodique
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -10106,29 +10102,31 @@ vegan:en:yes
+el:E370, 1,4-Επτανολακτόνη
+es:E370, 1,4-Heptonolactona
+et:E370, 1,4-Heptonolaktoon
+fi:E370, 1,4-Heptonolaktoni
```
```suggestion
fi:E370, 1\,4-Heptonolaktoni, 1\,4-Heptonolaktonia
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -10353,27 +10351,27 @@ wikidata:en:Q13081804
+el:E387, οξυστεαρίνη
+es:E387, oxistearina
+et:E387, oksüsteariin
+fi:E387, oksisteariini
```
```suggestion
fi:E387, Oksisteariini, Oksisteariinia
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -10438,29 +10436,30 @@ vegan:en:yes
+el:E390, Διστεαρυλοθειοδιπροπιονική
+es:E390, Tiodipropionato de distearilo
+et:E390, Distearyl thiodipropionate
+fi:E390, Distearyylitiodipropionaatti
```
```suggestion
fi:E390, Distearyylitiodipropionaatti, Distearyylitiodipropionaattia
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -10531,27 +10530,27 @@ vegetarian:en:yes
+el:E399, Λακτοβιονικό ασβέστιο
+es:E399, Lactobionato de calcio
+et:E399, Kaltsiumlaktobionaat
+fi:E399, Kalsiumlaktobionaatti
```
```suggestion
fi:E399, Kalsiumlaktobionaatti, Kalsiumlaktobionaattia
```

teolemon committed:
Apply suggestions from code review

teolemon commented on taxonomies/additives.txt:
```diff
@@ -8714,32 +8714,28 @@ wikidata:en:Q17997701
+es:E336, Tartratos de sodio
+et:E336, Naatriumtartraadid
+fi:E336, Natriumtartraatti, Natriumtartraattia, Natriumtartraatit
+fr:E336, Tartrates de sodium
```
```suggestion
fr:E336, Tartrates de potassium, Tartrate de potassium, argol, crème de tartre
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -9073,14 +9069,14 @@ wikipedia:en:https://en.wikipedia.org/wiki/Trisodium_phosphate
+da:E340, Kaliumfosfater
+de:E340, Kaliumphosphate
+el:E340, Ορθοφωσφορικά κάλια
+es:E340, Fosfatos de potasio
```
```suggestion
es:E340, Fosfatos de potasio, Fosfato de potasio, Fosfato potásico
```

teolemon committed:
Apply suggestions from code review

teolemon commented on taxonomies/additives.txt:
```diff
@@ -8714,32 +8714,28 @@ wikidata:en:Q17997701
+cs:E336, Sodné soli kyseliny vinné
+da:E336, Natriumtartrater
+de:E336, Natriumtartrate
+el:E336, Ταρτρικά αλάτα νατρίου
```
```suggestion
el:E336, Ταρτρικά αλάτα νατρίου
eo:E336, Kalia tartrato
```

teolemon committed:
Apply suggestions from code review

stephanegigandet commented on taxonomies/additives.txt:
```diff
@@ -8714,32 +8714,29 @@ wikidata:en:Q17997701
-de:E336, Kaliumtartrate
-el:E336, E336 food additive
+bg:E336, Натриеви тартарати
+ca:E336, Tartrats de sodi
```
All the added translations are for sodium tartrate (E335), not potassium tartrate (E336)

stephanegigandet commented on taxonomies/additives.txt:
```diff
@@ -10106,29 +10103,31 @@ vegan:en:yes
-el:E370, E370 food additive
-es:E370, E370 food additive
-et:E370, E370 food additive
+bg:E370, 1,4-Хептонолактон
```
the comma needs to be escaped

teolemon committed:
Merge branch 'main' into additives

teolemon commented:
/lint

Commit:
chore: Linting changes

teolemon commented on taxonomies/additives.txt:
```diff
@@ -10107,29 +10104,31 @@ vegan:en: yes
-sk: E370, E370 food additive
-sl: E370, E370 food additive
-sv: E370, E370 food additive
+fr: E370, 1,4-Heptonolactone
+hu: E370, 1,4-Heptonolakton
+it: E370, 1,4-Eptanolattone
+lt: E370, 1,4-Heptonolaktonas
+lv: E370, 1,4-Heptonolaktons
+mt: E370, 1,4-Heptonolakton
+nl: E370, 1,4-Heptonolacton
+pl: E370, 1,4-Heptonolakton
+pt: E370, 1,4-Heptonolactona
+ro: E370, 1,4-Heptonolactonă
+sk: E370, 1,4-Heptonolakton
+sl: E370, 1,4-Heptonolakton
+sv: E370, 1,4-Heptonolakton
```
```suggestion
fr: E370, 1\,4-Heptonolactone
hu: E370, 1\,4-Heptonolakton
it: E370, 1\,4-Eptanolattone
lt: E370, 1\,4-Heptonolaktonas
lv: E370, 1\,4-Heptonolaktons
mt: E370, 1,\4-Heptonolakton
nl: E370, 1\,4-Heptonolacton
pl: E370, 1\,4-Heptonolakton
pt: E370, 1\,4-Heptonolactona
ro: E370, 1\,4-Heptonolactonă
sk: E370, 1\,4-Heptonolakton
sl: E370, 1\,4-Heptonolakton
sv: E370, 1\,4-Heptonolakton
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -10107,29 +10104,31 @@ vegan:en: yes
-el: E370, E370 food additive
-es: E370, E370 food additive
-et: E370, E370 food additive
+bg: E370, 1,4-Хептонолактон
+cs: E370, 1,4-Heptonolakton
+da: E370, 1,4-Heptonolakton
+de: E370, 1,4-Heptonolacton
+el: E370, 1,4-Επτανολακτόνη
+es: E370, 1,4-Heptonolactona
+et: E370, 1,4-Heptonolaktoon
```
```suggestion
bg: E370, 1\,4-Хептонолактон
cs: E370, 1\,4-Heptonolakton
da: E370, 1\,4-Heptonolakton
de: E370, 1\,4-Heptonolacton
el: E370, 1\,4-Επτανολακτόνη
es: E370, 1\,4-Heptonolactona
et: E370, 1\,4-Heptonolaktoon
```

teolemon replied:
@stephanegigandet is the escaping good

teolemon committed:
Update taxonomies/additives.txt

teolemon committed:
Update taxonomies/additives.txt

teolemon commented:
/lint

stephanegigandet commented on taxonomies/additives.txt:
```diff
@@ -8715,32 +8715,29 @@ wikidata:en: Q17997701
-pl: E336, Winiany potasu
-pt: E336, Tartarato de potássio
-ro: E336, E336 food additive
-ru: E336, тартрат калия
```
removed?

teolemon commented on taxonomies/additives.txt:
```diff
@@ -8715,32 +8715,29 @@ wikidata:en: Q17997701
+nl: E336, Natriumtartraat, Natriumtartraten
+pl: E336, Winiany sodu, winian sodu, winian sodowy
+pt: E336, Tartratos de sódio
+ro: E336, Tartrati de sodiu
```
```suggestion
ro: E336, Tartrati de sodiu
ru: E336, тартрат калия
```

teolemon committed:
Update taxonomies/additives.txt

teolemon commented on taxonomies/additives.txt:
```diff
@@ -8715,32 +8715,30 @@ wikidata:en: Q17997701
-sl: E336, E336 food additive
-sr: E336, Kalijum tartrat
-sv: E336, Kaliumtartrater
+sk: E336, Sodné soli kyseliny vínovej
```
```suggestion
sh: E336, Kalijum tartrat
sk: E336, Sodné soli kyseliny vínovej
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -8715,32 +8715,30 @@ wikidata:en: Q17997701
-sr: E336, Kalijum tartrat
-sv: E336, Kaliumtartrater
+sk: E336, Sodné soli kyseliny vínovej
+sl: E336, Natrijevi tartrati
```
```suggestion
sl: E336, Natrijevi tartrati
sr: E336, Kalijum tartrat
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -9074,14 +9072,14 @@ wikipedia:en: https://en.wikipedia.org/wiki/Trisodium_phosphate
-de: E340, Kaliumphosphat, Kaliumphosphate
-el: E340, E340 food additive
+bg: E340, Калиеви фосфати
+ca: E340, Fosfats de potassi
```
```suggestion
ca: E340, Fosfat de potassi, fosfats de potassi
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -9971,27 +9969,27 @@ wikidata:en: Q213050
+el: E364, Φουμαρικά νάτρια
+es: E364, Succinatos de sodio
+et: E364, Naatriumsuktsinaadid
+fi: E364, Natriumsukkinaatit
```
```suggestion
fi: E364, Natriumsukkinaatit, Natriumsukkinaattia
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -11225,29 +11227,31 @@ vegan:en: yes
+el: E414a, Τροποποιημένη αραβική κόμμι με οκτενυλοσουκκινικό οξύ
+es: E414a, Goma arábiga modificada con ácido octenilsuccínico
+et: E414a, Oktenüülsuktsiinhappega modifitseeritud araabia kumm
+fi: E414a, Oktenyylisukkinihappoa muunnettu arabikumi
```
```suggestion
fi: E414a, Oktenyylimeripihkahappo, muunnettu arabikumi, Oktenyylimeripihkahappoa, muunnettua arabikumia, Oktenyylisukkinihappoa muunnettu arabikumi
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -11403,27 +11407,27 @@ wikidata:en: Q416694
+el: E419, Γκάτι κόμμι
+es: E419, Goma ghatti
+et: E419, Ghatti kummi
+fi: E419, Ghattikumi
```
```suggestion
fi: E419, Ghatti-kumi, Ghatti-kumia, Ghattikumi
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -11969,29 +11973,31 @@ vegetarian:en: yes
+el: E430, Πολυοξυαιθυλενιο (8) στεαρικό
+es: E430, Polioxietileno (8) estearato
+et: E430, Polüoksüetüleen (8) stearaat
+fi: E430, Polyoksyetyleeni (8) stearaatti
```
```suggestion
fi: E430, Polyoksyetyleeni (8) stearaatti, Polyoksietyleeni(8)monostearaatti, Polyoksietyleeni(8)monostearaattia
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -12248,27 +12254,27 @@ wikidata:en: Q2103107
+fi: E440a, Pektiini
 fr: E440a, Pectine
-hr: E440a, pektin, pektini, tvar za želiranje pektin, zgušnjivač pektini
+hr: E440a, Pektin
```
```suggestion
hr: E440a, Pektin, pektini, tvar za želiranje pektin, zgušnjivač pektini
```

teolemon committed:
Apply suggestions from code review

teolemon commented:
/lint

teolemon commented:
/lint

teolemon committed:
Merge branch 'main' into additives

teolemon commented:
/lint

Commit:
chore: Linting changes

teolemon committed:
Merge branch 'main' into additives

teolemon committed:
Merge branch 'main' into additives

teolemon committed:
Merge branch 'main' into additives

teolemon committed:
Merge branch 'main' into additives

Copilot committed:
taxonomy: fix additive translation review issues

Agent-Logs-Url: https://github.com/openfoodfacts/openfoodfacts-server/sessions/7ca5b1e7-9add-42b4-b9f6-88b734804b59

Co-authored-by: teolemon <1689815+teolemon@users.noreply.github.com>

teolemon committed:
Merge branch 'main' into additives

teolemon committed:
Merge branch 'main' into additives

teolemon commented on taxonomies/additives.txt:
```diff
@@ -11492,22 +11553,50 @@ wikidata:en: Q421991
+lv: E408, E408 food additive
+mt: E408, E408 food additive
+nl: E408, E408 food additive
+pl: E408, E408 food additive
```
```suggestion
```

teolemon replied:
```suggestion
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -11561,10 +11650,27 @@ wikidata:en: Q221391
+el: E411, Βρώμης γόμα
+es: E411, Goma de avena
+et: E411, Kaerahelbe kumm
+fi: E411, Kaurakumi
```
```suggestion
fi: E411, Kaurakumi, Kaurakumia
```

teolemon committed:
Update additives.txt

teolemon committed:
Merge branch 'additives' of https://github.com/openfoodfacts/openfoodfacts-server into additives

teolemon commented on taxonomies/additives.txt:
```diff
@@ -9486,11 +9486,15 @@ en: E340, Potassium phosphates, potassium phosphate, E 340, E-340
-fr: E340, Orthophosphates de potassium, Orthophosphate de potassium, Phosphate de potassium, Phosphates de potassium
-hr: E340, kalijev fosfat, kalijevi fosfati
+et: E340, Kaaliumfosfaadid
+fi: E340, Kaliumfosfaatti, Kaliumfosfaattia
```
```suggestion
fi: E340, Kaliumfosfaatti, Kaliumfosfaattia, kaliumfosfaatit
```

Commit:
chore: Linting changes

teolemon committed:
Update additives.txt

teolemon committed:
Update additives.txt

teolemon commented on taxonomies/additives.txt:
```diff
@@ -9486,11 +9486,15 @@ en: E340, Potassium phosphates, potassium phosphate, E 340, E-340
 fi: E340, Kaliumfosfaatti, Kaliumfosfaattia, kaliumfosfaatit
 fr: E340, Orthophosphates de potassium, Orthophosphate de potassium, Phosphate de potassium, Phosphates de potassium
-hr: E340, kalijev fosfat, kalijevi fosfati
+hr: E340, kalijev fosfat
```
```suggestion
hr: E340, kalijev fosfat, kalijevi fosfati
```

teolemon commented on taxonomies/additives.txt:
```diff
@@ -10038,33 +10045,32 @@ wikidata:en: Q45044
-fr: E349, Malate d'ammonium
-hu: E349, Ammónium-malát
-it: E349, Malato di ammonio
+fr: E349, Malates d'ammonium
```
```suggestion
fr: E349, Malates d'ammonium, Malate d'ammonium
```

teolemon committed:
Update additives.txt

Freso commented on taxonomies/additives.txt:
```diff
@@ -9316,8 +9316,12 @@ en: E340, Potassium phosphates, potassium phosphate, E 340, E-340
-de: E340, Kaliumphosphat, Kaliumphosphate
+cs: E340, Fosfáty draselné
+da: E340, Kaliumfosfater
+de: E340, Kaliumphosphate
```
is “Kaliumphosphat” outright wrong, or why are you removing this?

Freso commented on taxonomies/additives.txt:
```diff
@@ -9858,33 +9865,32 @@ wikidata:en: Q45044
-de: E349, Ammoniummalat
-el: E349, Μηλικό αμμώνιο
-eo: E349, Amonia malato
-es: E349, Malato de amonio, Malato amónico, E 349
```
Were the previous ones bad/wrong? Or why are they not kept as synonyms?

Freso replied:
is “E 349” getting caught (ie., translated to “E349”) by the parser already? or should it be added to `xx`?

Freso commented on taxonomies/additives.txt:
```diff
@@ -9858,33 +9865,32 @@ wikidata:en: Q45044
-pl: E349, Jabłczan amonu
-pt: E349, Malato de amónio
-ro: E349, Malat de amoniu
+lt: E349, Amonio malatai
```
-ai is the plural form, so from the little lithuanian i know, i'm guessing “amonio malatas” is the proper singular form here and should probably be kept as a synonym.

Freso reviewed (commented):
A lot of translations/synonyms get dropped entirely, and it isn’t clear if it’s because they’re _wrong_ or if there’s some other reason for dropping them.

aleene commented:
You should distinguish between singular and plural. At the top level it should be plural. The children specify the variant, i.e. singular. Often the various children have a different number of Hydrogen-atoms. The parent indicates them all. Might depend on language.

Maybe this explains the edits (did not check).
